{"metadata":{"title":"PublicKeyCredentialError.general(message:)","role":"symbol","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"general","kind":"identifier"},{"kind":"text","text":"("},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"modules":[{"name":"FIDO2"}],"externalID":"s:5FIDO224PublicKeyCredentialErrorO7generalyACSS_tcACmF","symbolKind":"case"},"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError"]]},"abstract":[{"text":"General error with custom message.","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":")"}],"languages":["swift"]}]}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/general(message:)"},"variants":[{"paths":["\/documentation\/fido2\/publickeycredentialerror\/general(message:)"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError":{"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError","title":"PublicKeyCredentialError","type":"topic","url":"\/documentation\/fido2\/publickeycredentialerror","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"PublicKeyCredentialError"}],"role":"symbol","navigatorTitle":[{"text":"PublicKeyCredentialError","kind":"identifier"}],"abstract":[{"text":"A type that indicates when Fido2ApiClient encounters an error.","type":"text"}],"kind":"symbol"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2":{"role":"collection","type":"topic","kind":"symbol","abstract":[],"url":"\/documentation\/fido2","title":"FIDO2","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError/general(message:)":{"abstract":[{"type":"text","text":"General error with custom message."}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/general(message:)","url":"\/documentation\/fido2\/publickeycredentialerror\/general(message:)","title":"PublicKeyCredentialError.general(message:)","role":"symbol","kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"general","kind":"identifier"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"type":"topic"}}}